### DISCUSSION TITLE
Is it possible to group lots of options like  zsh?

### DISCUSSION THREAD

**Discussion by CarterLi** [carapace-sh/carapace-bin#2053] [Q&A] [answered]

**Comment by rsteube** [comment, marked as answer]
[Commands](https://rsteube.github.io/carapace/carapace/command/group.html) and [arguments](https://rsteube.github.io/carapace/carapace/action/tag.html), but not flags.

**Reply by CarterLi** [reply]
It should be useful if gets supported
